fe/fe-core/src/test/java/com/starrocks/load/ExportCheckerTest.java

```diff
@@ -49,13 +49,6 @@ public Backend getBackend(long backendId) {
             }
         };
 
-        new MockUp<Backend>() {
```

Comment:
does this mockup only affect current case or any other case?

Reply:
It seems like it affect other case, check this: 
```
17:28:50 [ERROR] testSchemaChange1(com.starrocks.alter.RollupJobV2Test)  Time elapsed: 7.121 s  <<< ERROR!
17:28:50 Missing 1 invocation to:
17:28:50 com.starrocks.system.Backend#isAlive()
17:28:50    on mock instance: com.starrocks.system.Backend@78f2029d
17:28:50 Caused by: Missing invocations
17:28:50         at com.starrocks.system.Backend.isAlive(Backend.java)
17:28:50         at com.starrocks.task.AgentBatchTask.run(AgentBatchTask.java:161)
17:28:50         at java.lang.Thread.run(Thread.java:750)
```